.github/workflows/ruby.yml

```diff
@@ -17,32 +17,32 @@ jobs:
         rails_version: ["8.0.1"]
         blacklight_version: ["~> 8.0"]
         experimental: [false]
-        additional_engine_cart_rails_options: ["--css bootstrap"]
+        additional_engine_cart_rails_options: [""] # default: propshaft, cssbundling, importmaps
```

Comment:
isn't "bootstrap" the default? https://github.com/projectblacklight/blacklight-gallery/pull/195/files#diff-ee98e028c59b193d58fde56ab4daf54d43c486ae674e63d50ddf300b07943e0fR16

Reply:
I was misunderstanding what cssbundling does, it just provides the installers. I'll update the comment.